examples/examples/contract_instantiate_with_code.rs

```diff
@@ -89,6 +89,8 @@ async fn main() {
 	let xt = api.contract_call(contract.into(), 500_000, 500_000, vec![0u8]);
 
 	println!("[+] Calling the contract with extrinsic Extrinsic:\n{:?}\n\n", xt);
-	let report = api.submit_and_watch_extrinsic_until(xt, XtStatus::Finalized).unwrap();
+	let report = api
```

Comment:
Leave as is: Normal users want to know if xt was successful or not. Same goes for all the examples I believe.

Reply:
`api.submit_and_watch_extrinsic_until` is now `api.submit_and_watch_extrinsic_until_without_events`.
`api.submit_and_watch_extrinsic_until` now fetch the events

Reply:
Yes, I know. But this is an example for the users - so we should use what we believe is best practice. And personally, I'd recommend to use `submit_and_watch_extrinsic_until`.

Reply:
I don't think we should hide `submit_and_watch_extrinsic_until_without_events`. Why should a user call `submit_and_watch_extrinsic_until` if he is not interested in events?
In any case, we should rethink this in another problem: #591

Reply:
It's not just about events, but mostly about the Error part - `submit_and_watch_extrinsic_until_without_events` will not error, even if the extrinsic has failed onchain. This is fairly misleading for the unaware user.

But we can do that in #591, agreed.